providers/amazon/src/airflow/providers/amazon/aws/log/cloudwatch_task_handler.py

```diff
@@ -206,8 +197,6 @@ def get_cloudwatch_logs(self, stream_name: str, task_instance: RuntimeTI):
             log_stream_name=stream_name,
             end_time=end_time,
         )
-        if AIRFLOW_V_3_0_PLUS:
-            return list(self._event_to_dict(e) for e in events)
```

Comment:
@jason810496 This line means that logs from Cloudwatch are now showing as raw-json strings rather then rendered nicely. What was the thinking here?

Reply:
@jason810496 A sort of fix (but not a complete one) that I knocked up https://github.com/astronomer/airflow/commit/afbf6f514be23b671225ccc4d275a76d77e04989

Could you finish it off and get logs for cloudwatch working properly please.

I was using https://www.localstack.cloud/ (the free version) and I set these env vars in breeze to test it

```
AWS_ENDPOINT_URL=http://localstack-main:4566
AWS_ACCESS_KEY_ID="test" AWS_SECRET_ACCESS_KEY="test"
AWS_DEFAULT_REGION="us-east-1 
AIRFLOW__LOGGING__REMOTE_BASE_LOG_FOLDER=cloudwatch://arn:aws:logs:us-east-1:000000000000:log-group:test
AIRFLOW__LOGGING__REMOTE_LOGGING=True
```